templates/legal/shared/_uasd-toc-ja.html

```diff
@@ -17,4 +17,10 @@ <h3>追加情報</h3>
     <li class="p-nested-counter-list__item"><a href="#uasd-ua-support-services">Ubuntuアドバンテージのサポート<br/>サービスプロセス</a></li>
     <li class="p-nested-counter-list__item"><a href="#uasd-definitions">用語の定義</a></li>
   </ol>
+</nav>
+<nav class="p-card">
+  <h3>English translation</h3>
```

Comment:
it isn't a translation... version?
```suggestion
  <h3>English version</h3>
```